demos/demos.scss

```diff
@@ -14,16 +14,17 @@
   limitations under the License
 */
 
-@import "https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.min.css";
-
+// @import "https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.min.css";
 
 /* Base style */
 
-:root {
-  --mdc-theme-primary: #212121;
-  --mdc-theme-accent: #64DD17;
-}
+$mdc-theme-primary: #212121;
+$mdc-theme-accent: #64dd17;
 
+@import "@material/theme/mdc-theme";
```

Comment:
Why not just `@import 'material-components-web/material-components-web'` here and get rid of the import within the demo files?